ddtrace/contrib/redis/patch.py

```diff
@@ -4,14 +4,20 @@
 from ddtrace import config
 from ddtrace.vendor import wrapt
 
+from ...internal.schema import span_attribute_schema
```

Comment:
This is a bit of magic; I'm importing an instantiated instance of the schema version class.  Is this ok?

Reply:
Checked with team: 
From Brett: "If we can avoid global instances let's do it.
Is it possible that we don't need a class/instance for implementing that?"

Reply:
I believe I can make them functions instead of classes/objects